python/tskit/trees.py

```diff
@@ -3846,40 +3890,62 @@ class for more details on how missing data is reported.
                 genotypes = bytes_genotypes.tobytes()
             yield Variant(site, alleles, genotypes)
 
-    def genotype_matrix(self, impute_missing_data=False, alleles=None):
+    def genotype_matrix(
+        self, *, isolated_as_missing=None, alleles=None, impute_missing_data=None
```

Comment:
I'm not sure ``impute_missing_data`` for the genotype matrix was released either, worth having a check.

Reply:
This was in 0.2.3 also https://github.com/tskit-dev/tskit/blob/0.2.3/python/tskit/trees.py#L2936